### PR TITLE
EPMLSTRCMW-234: Migrations fail to execute when started from docker

### PR DESCRIPTION
**Pull request by iliaonishchenko**
These MR contains two major updates:

- Fix liquibase migrations to make possible to start pipeline via docker-compose.

- Refine docker-compose to make it more fault tolerant.

### CODE REVIEW

**Review by vasp1ru — APPROVED** [review]

**Review by gominasian — APPROVED** [review]
Approved even from two accounts)

File: build.sbt [hunk 1]
```diff
@@ -28,7 +28,8 @@ lazy val commonSettings = Seq(
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-h", "target/test-reports"),
   coverageEnabled in Test := true,
   coverageMinimum := 60,
-  coverageFailOnMinimum := true
+  coverageFailOnMinimum := true,
+  coverageExcludedPackages := "cromwell.pipeline.database.*"
```
Comment (Nikolai-Isaev) [inline build.sbt:32]: Shouldn't we put an asterisk before the path name? Or sbt will find that database package anyway using just a piece of relative path?

**Review by Nikolai-Isaev — APPROVED** [review]
Good to me

**Review by DruzhininPavel — APPROVED** [review]
Let`s rock!

**Review by GrigoriiBerezin — APPROVED** [review]
Nice

File: build.sbt (hunk 1 above)
Comment (iliaonishchenko) [reply build.sbt:32]: It looks like sbt is able to find like this.
Before I added this line there was coverage fail :)

**Review by Dmitrii-Shevchenko — APPROVED** [review]

**Comment by Dmitrii-Shevchenko** [comment]
Please remove postgres-mongo.yaml and correct the readme associated with this file

**Comment by iliaonishchenko** [comment]
@Dmitrii-Shevchenko, I've added requested changes, please take a look.

**Pull request merged**